.cirrus.yml

```diff
@@ -597,7 +597,6 @@ docker_builder:
     - tool_tests_integration-linux
     - build_tests-linux
     - integration_tests-linux
-    - integration_tests_gradle-linux
```

Comment:
Do we want this to now depend on integration_tests_gradle1-linux and integration_tests_gradle2-linux?